lib/ansible/modules/net_tools/nios/nios_a_record.py

```diff
@@ -116,6 +116,17 @@
       password: admin
   connection: local
 
+- name: update an A record IP address
+  nios_a_record:
+    name: a.ansible.com
+    ipv4: {new_ipv4addr: 10.10.10.1, old_ipv4addr: 192.168.10.1}
```

Comment:
Plz comment the new param added in module doc (ref: [nios_host_record](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/net_tools/nios/nios_host_record.py#L31)).

Reply:
Done for both nios_a_record and nios_txt_record